src/ExpressiveInstaller/Resources/src/CallableFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Interop\Container\ContainerInterface;
```

Comment:
There is #160 to change to `Psr\Container\ContainerInterface`. As soon as I have some spare time I'll merge it.